gateway/src/cluster/builder.rs

```diff
@@ -127,6 +130,40 @@ impl ClusterBuilder {
         self
     }
 
+    /// Set the properties for shards to identify with.
+    ///
+    /// This may be used if you want to set a different operating system.
```

Comment:
This wording seems a bit strong, like this can *only* be used to change the identifying OS. Is that intentional?